src/DotNet.Status.Web/Controllers/TelemetryController.cs

```diff
@@ -0,0 +1,78 @@
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Microsoft.DotNet.Kusto;
+using Kusto.Ingest;
+using System.Collections.Generic;
+
+namespace DotNet.Status.Web.Controllers
+{
+    [Route("api/telemetry")]
+    [ApiController]
+    public class TelemetryController : ControllerBase
+    {
+        private readonly ILogger<TelemetryController> _logger;
+        private readonly IOptionsSnapshot<TelemetryOptions> _options;
+        private readonly IKustoIngestClient _client;
+
+        public TelemetryController(
+            ILogger<TelemetryController> logger,
+            IOptionsSnapshot<TelemetryOptions> options,
+            IKustoIngestClient client = null)
+        {
+            _logger = logger;
+            _options = options;
+            _client = client;
+        }
+
+        [HttpPost("collect/ArcadeValidation")]
+        public async Task<IActionResult> CollectArcadeValidation([Required] ArcadeValidationData data)
+        {
+            _logger.LogInformation("Start Collect Arcade Validation data");
+
+            TelemetryOptions options = _options.Value;
+
+            if(null == options)
```

Comment:
Please don't use yoda conditionals. you can use `options is null` or `options == null` whichever you prefer.

Reply:
Arrrggg... been writing too much Powershell.

Reply:
Yea, sadly powershell can have different semantics for each order, I prefer `if ($a)` in powershell to avoid all that.